docs/user-manual/modules/ROOT/pages/index.adoc

```diff
@@ -533,6 +534,53 @@ camel routes without them knowing
 * xref:team.adoc[Team]
 * https://github.com/camel-extra/camel-extra[Camel Extra]
 
+== Requirements
+
+=== Platform Requirements
+
+* Any platform that supports JDK 6 is required.
+
+=== JDK Requirements
+
+* Camel 2.6 or older requires JDK 1.5 or better.
+
+* Camel 2.7 onwards requires JDK 1.6 or JDK 1.7.
+
+* Camel 2.14 onwards requires JDK 1.7 or better.
+
+* Camel 2.18 onwards requires JDK 1.8
+
+=== Camel Jar Requirements
```

Comment:
I think this section could be completely removed. The other related to JDK could be useful.

Reply:
oh okay ,I'll do that